mail/postfix/src/opnsense/service/templates/OPNsense/Postfix/main.cf

```diff
@@ -43,9 +43,9 @@ smtp_tls_CAfile = /etc/ssl/cert.pem
 ##########################
 
 {% if helpers.exists('OPNsense.postfix.general.myhostname') and OPNsense.postfix.general.myhostname != '' %}
-myhostname = {{ OPNsense.postfix.general.myhostname }}
+myhostname = {{ OPNsense.postfix.general.myhostname }}.{{ OPNsense.postfix.general.mydomain }}
```

Comment:
But this one is too fragile as mydomain could be empty

Reply:
I think we should leave this and let people use xxx.$mydomain if they want to (it may require adjusting the validation)

Reply:
Yes, makes more sense